carfield.mk

```diff
@@ -97,7 +104,7 @@ include $(CAR_ROOT)/utils/venv.mk
 
 QUESTA_FLAGS := -permissive -suppress 3009 -suppress 8386 -error 7
 ifdef DEBUG
-	VOPT_FLAGS := $(QUESTA_FLAGS) +acc
+	VOPT_FLAGS := +acc $(QUESTA_FLAGS)
```

Comment:
why this?

Reply:
It remained in the buffer, I fixed it. Sorry!